C# template doesn't compile

<!-- Thanks for taking the time to report an issue!  Some things to consider:

 - Please be detailed and ensure there isn't already an open issue for your bug/request.
 - If you can, please try to reproduce the problem using the develop branch first as your issue may already be fixed.
 - If you have an enhancement request, you can remove the sections that don't apply.
 
-->

## Expected Behavior

Projects generated by `dotnet new etoapp` compile. 

## Actual Behavior

`dotnet build` fails due to the error "error CS0246: The type or namespace name 'Form' could not be found" in `MainForm.cs`.

It seems to me that this behavior is caused by 1c353b7, which removes some `using`s in favor of global `using`. The commit also removed the `using` declarations from the template code, but it seems to me that neither the nuget nor the .csproj files in the template sets up global `using`s for the code. 

To be clear, I was not troubled with making the generated compile itself; I can add `using Eto.Forms` and some other `using`s in the generated .cs files to remove the errors. 

## Steps to Reproduce the Problem

  1. `dotnet new install Eto.Forms.Templates` to install the latest template (2.8.2). 
  2. `mkdir SomeProj` and `cd SomeProj`
  3. `dotnet new etoapp`
  4. `dotnet build SomeProj`, which fails with the above-mentioned error. 

## Specifications

  - Version: 2.8.2
  - Platform(s): Mac 
  - Operating System(s): macOS 14.1

--- a/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/MainForm.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/MainForm.cs
@@ -1,4 +1,8 @@
 #if (UseCode || UseCodePreview)
+using System;
+using Eto.Forms;
+using Eto.Drawing;
+
 namespace EtoApp._1
 {
 	public partial class MainForm : Form

--- a/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/MainForm.eto.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/MainForm.eto.cs
@@ -1,4 +1,8 @@
 #if (UseCodePreview)
+using System;
+using Eto.Forms;
+using Eto.Drawing;
+
 namespace EtoApp._1
 {
 	partial class MainForm : Form

--- a/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/MainForm.jeto.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/MainForm.jeto.cs
@@ -1,4 +1,8 @@
 ﻿#if (UseJeto)
+using System;
+using System.Collections.Generic;
+using Eto.Forms;
+using Eto.Drawing;
 using Eto.Serialization.Json;
 
 namespace EtoApp._1

--- a/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/MainForm.xeto.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/MainForm.xeto.cs
@@ -1,4 +1,8 @@
 ﻿#if (UseXeto)
+using System;
+using System.Collections.Generic;
+using Eto.Forms;
+using Eto.Drawing;
 using Eto.Serialization.Xaml;
 
 namespace EtoApp._1

--- a/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/EtoApp.1/Program.cs
@@ -1,4 +1,8 @@
-﻿namespace EtoApp._1
+﻿using System;
+using Eto.Forms;
+using Eto.Drawing;
+
+namespace EtoApp._1
 {
 	class Program
 	{

--- a/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.Gtk/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.Gtk/Program.cs
@@ -1,4 +1,7 @@
-﻿namespace EtoApp._1.Gtk
+﻿using System;
+using Eto.Forms;
+
+namespace EtoApp._1.Gtk
 {
 	class Program
 	{

--- a/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.Mac/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.Mac/Program.cs
@@ -1,4 +1,7 @@
-﻿namespace EtoApp._1.Mac
+﻿using System;
+using Eto.Forms;
+
+namespace EtoApp._1.Mac
 {
 	class Program
 	{

--- a/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.MacOS/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.MacOS/Program.cs
@@ -1,4 +1,7 @@
-﻿namespace EtoApp._1.Mac
+﻿using System;
+using Eto.Forms;
+
+namespace EtoApp._1.Mac
 {
 	class Program
 	{

--- a/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.WinForms/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.WinForms/Program.cs
@@ -1,4 +1,7 @@
-﻿namespace EtoApp._1.WinForms
+﻿using System;
+using Eto.Forms;
+
+namespace EtoApp._1.WinForms
 {
 	class Program
 	{

--- a/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.Wpf/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.Wpf/Program.cs
@@ -1,4 +1,7 @@
-﻿namespace EtoApp._1.Wpf
+﻿using System;
+using Eto.Forms;
+
+namespace EtoApp._1.Wpf
 {
 	class Program
 	{

--- a/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.XamMac/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-CSharp/Separate/EtoApp.1.XamMac/Program.cs
@@ -1,4 +1,6 @@
 ﻿using AppKit;
+using Eto.Forms;
+
 namespace EtoApp._1.XamMac
 {
 	class Program

--- a/src/Eto.Forms.Templates/content/App-FSharp/Separate/EtoApp.1.Wpf/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-FSharp/Separate/EtoApp.1.Wpf/Program.cs
@@ -1,4 +1,7 @@
-﻿namespace EtoApp._1.Wpf
+﻿using System;
+using Eto.Forms;
+
+namespace EtoApp._1.Wpf
 {
 	class Program
 	{

--- a/src/Eto.Forms.Templates/content/App-VisualBasic/Separate/EtoApp.1.MacOS/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-VisualBasic/Separate/EtoApp.1.MacOS/Program.cs
@@ -1,4 +1,7 @@
-﻿namespace EtoApp._1.Mac
+﻿using System;
+using Eto.Forms;
+
+namespace EtoApp._1.Mac
 {
 	class Program
 	{

--- a/src/Eto.Forms.Templates/content/App-VisualBasic/Separate/EtoApp.1.XamMac/Program.cs
+++ b/src/Eto.Forms.Templates/content/App-VisualBasic/Separate/EtoApp.1.XamMac/Program.cs
@@ -1,4 +1,6 @@
 ﻿using AppKit;
+using Eto.Forms;
+
 namespace EtoApp._1.XamMac
 {
 	class Program

--- a/src/Eto.Forms.Templates/content/File-CSharp/MainForm.cs
+++ b/src/Eto.Forms.Templates/content/File-CSharp/MainForm.cs
@@ -1,4 +1,8 @@
 #if (UseCode || UseCodePreview)
+using System;
+using Eto.Forms;
+using Eto.Drawing;
+
 namespace EtoApp._1
 {
 	public partial class MainForm : Form

--- a/src/Eto.Forms.Templates/content/File-CSharp/MainForm.eto.cs
+++ b/src/Eto.Forms.Templates/content/File-CSharp/MainForm.eto.cs
@@ -1,4 +1,8 @@
 #if (UseCodePreview)
+using System;
+using Eto.Forms;
+using Eto.Drawing;
+
 namespace EtoApp._1
 {
 	partial class MainForm : Form

--- a/src/Eto.Forms.Templates/content/File-CSharp/MainForm.jeto.cs
+++ b/src/Eto.Forms.Templates/content/File-CSharp/MainForm.jeto.cs
@@ -1,4 +1,8 @@
 ﻿#if (UseJeto)
+using System;
+using System.Collections.Generic;
+using Eto.Forms;
+using Eto.Drawing;
 using Eto.Serialization.Json;
 
 namespace EtoApp._1

--- a/src/Eto.Forms.Templates/content/File-CSharp/MainForm.xeto.cs
+++ b/src/Eto.Forms.Templates/content/File-CSharp/MainForm.xeto.cs
@@ -1,4 +1,8 @@
 ﻿#if (UseXeto)
+using System;
+using System.Collections.Generic;
+using Eto.Forms;
+using Eto.Drawing;
 using Eto.Serialization.Xaml;
 
 namespace EtoApp._1

